internal/http/services/owncloud/ocdav/proppatch.go

```diff
@@ -217,6 +218,17 @@ func (s *svc) handleProppatch(ctx context.Context, w http.ResponseWriter, r *htt
 						return nil, nil, false
 					}
 					currentUser := ctxpkg.ContextMustGetUser(ctx)
+					ok, err := utils.CheckPermission(ctx, "Favorites.Write", client)
```

Comment:
Can we use the constant WriteFavorites from a pkg/permission/permission.go instead of the "Favorites.Write"?

Reply:
utils.CheckPermission(ctx, permission.WriteFavorites, client)

Reply:
Excellent idea :+1: I did it for other occurrences of `CheckPermission` too. Please double check.